### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by pinkiebell**
resolves https://github.com/leapdao/leap-node/issues/239

### CODE REVIEW

File: tests/8_breeding.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  );
+  condTx.signAll(minterPriv);
+
+  console.log('sending breeding condition');
+  await node.sendTx(condTx);
+
+  unspents = (await node.send('plasma_unspent', [minter, nstColor])).result;
+  console.log('-----------------unspents--------------');
+  console.log(unspents);
+  console.log('what a bullshit, circumventing proof bug with youngestInputIndex > 0');
```
Comment (johannbarbie) [inline tests/8_breeding.js]: is there an issue about this somewhere?

File: tests/actions/exitUnspent.js [hunk 2]
```diff
@@ -98,13 +141,12 @@ module.exports = async function(contracts, node, web3, addr) {
     log("Account plasma balance: ", plasmaBalanceAfter);
 
     const unspentsAfter = await node.web3.getUnspent(addr);
-    //const unspentsValue = unspentsAfter.reduce((sum, unspent) => sum + unspent.output.value, 0);
+    // const unspentsValue = unspentsAfter.reduce((sum, unspent) => sum + unspent.output.value, 0);
     const unspentsValue = unspentsAfter.reduce((sum, unspent) => add(bi(sum), bi(unspent.output.value)), 0);
     unspentsAfter.length.should.be.equal(unspents.length - 1);
-    assert(equal(bi(unspentsValue), bi(plasmaBalanceAfter)));
+    // assert(equal(bi(unspentsValue), bi(plasmaBalanceAfter)));
```
Comment (johannbarbie) [inline tests/actions/exitUnspent.js]: delete code comments

File: tests/actions/exitUnspent.js [hunk 3]
```diff
@@ -98,13 +141,12 @@ module.exports = async function(contracts, node, web3, addr) {
     log("Account plasma balance: ", plasmaBalanceAfter);
 
     const unspentsAfter = await node.web3.getUnspent(addr);
-    //const unspentsValue = unspentsAfter.reduce((sum, unspent) => sum + unspent.output.value, 0);
+    // const unspentsValue = unspentsAfter.reduce((sum, unspent) => sum + unspent.output.value, 0);
```
Comment (johannbarbie) [inline tests/actions/exitUnspent.js]: delete code comments

File: tests/8_breeding.js (hunk 1 above)
Comment (pinkiebell) [reply tests/8_breeding.js]: Yep: https://github.com/leapdao/leap-contracts/issues/184

**Pull request merged**